content/api/commands/invoke.md

```diff
@@ -26,15 +35,19 @@ subject, use [`.its()`](/api/commands/its).
 **<Icon name="check-circle" color="green"></Icon> Correct Usage**
 
 ```javascript
-cy.wrap({ animate: fn }).invoke('animate') // Invoke the 'animate' function
+cy.get('.input').invoke('val').should('eq', 'foo') // Invoke the 'val' function
 cy.get('.modal').invoke('show') // Invoke the jQuery 'show' function
+cy.wrap({ animate: fn }).invoke('animate') // Invoke the 'animate' function
 ```
 
 **<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
 
 ```javascript
 cy.invoke('convert') // Errors, cannot be chained off 'cy'
 cy.wrap({ name: 'Jane' }).invoke('name') // Errors, 'name' is not a function
+cy.wrap({ animate: fn })
+  .invoke('animate')
+  .then(() => {}) // 'animate' will be called multiple times
```

Comment:
.then() is one of the few commands that doesn't actually doesn't retry (unless that has changed) so we might want to update this example

Reply:
I actually switched to `.then()` exactly for this reason - even if the following command doesn't retry and doesn't care about the subject, `.invoke()` will *still* get called twice - once when executing it, and then a second time when determining the subject for `.then()`.